rados/errors.go

```diff
@@ -12,27 +12,27 @@ import (
 	"github.com/ceph/go-ceph/internal/errutil"
 )
 
-// revive:disable:exported Temporarily live with stuttering
+// radosError represents an error condition returned from the Ceph RADOS APIs.
+type radosError int
 
-// RadosError represents an error condition returned from the Ceph RADOS APIs.
-type RadosError int
-
-// revive:enable:exported
-
-// Error returns the error string for the RadosError type.
-func (e RadosError) Error() string {
+// Error returns the error string for the radosError type.
+func (e radosError) Error() string {
 	errno, s := errutil.FormatErrno(int(e))
 	if s == "" {
 		return fmt.Sprintf("rados: ret=%d", errno)
 	}
 	return fmt.Sprintf("rados: ret=%d, %s", errno, s)
 }
 
+func (e radosError) Errno() int {
```

Comment:
This is newly added and not used anywhere. We can add a test case for this, may be in a different PR.